src/app/map/main-map/main-map.component.spec.ts

```diff
@@ -3,22 +3,49 @@ import { RouterTestingModule } from '@angular/router/testing';
 import { HttpClient, HttpHandler } from '@angular/common/http';
 import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
 import { CookieService } from 'ngx-cookie-service';
-import { LeafletMapComponent } from 'app/map/leaflet-map/leaflet-map.component';
 import { ProjectService } from 'app/services/project.service';
 import { MainMapComponent } from 'app/map/main-map/main-map.component';
 import { Project } from 'app/models/project';
+import { SearchComponent } from '../search/search.component';
```

Comment:
I believe all imports in this project are absolute. Should probably update these to match.